lib.php

```diff
@@ -1176,6 +1176,56 @@ function facetoface_get_attendees($sessionid) {
     return $records;
 }
 
+/**
+ * Get the user's stream from the organisation table.
+ *
+ * @param int $userid The user's ID.
+ * @return string|null The stream name or null if not found.
+ */
+function get_user_stream($userid): ?string {
+    global  $DB;
+
+    $sql = "SELECT 
+                toma.paydiv1name
+            FROM 
+                {tool_organisation_mtda_assi} toma
+            JOIN 
+                {tool_organisation_assign} toa 
+            ON toma.assignid = toa.id
+            JOIN 
+               {user} u 
+            ON u.id = toa.userid
+            WHERE 
+                u.id = :userid";
+
+    return $DB->get_field_sql($sql, ['userid' => $userid]);
+}
+
+/**
+ * Get the user's division from the organisation table.
+ *
+ * @param int $userid The user's ID.
+ * @return string|null The division name or null if not found.
+ */
+function get_user_division($userid): ?string {
+    global  $DB;
+
+    $sql = "SELECT 
+                tomp.division2name
+            FROM 
+                {tool_organisation_mtda_pos} tomp
+            JOIN 
+                {tool_organisation_assign} toa 
+            ON tomp.positionid = toa.positionid
+            JOIN 
+                {user} u 
+            ON u.id = toa.userid
+            WHERE 
+                u.id = :userid";
```

Comment:
The formatting on these queries seems a little bit off with the keywords being on their own lines.

The table ordering also seems somewhat backwards. I think it makes more sense to work from the table you're filtering on, `user` in this case, then join to the data that you want which would be `tool_organisation_mtda_pos` here.
Since `tool_organisation_assign` has a `userid` column and you're not using any fields from the `user` table, you can probably just make `tool_organisation_assign` the main `FROM` table and ignore the `user` table for optimisation.